resources/saas-boost-web.yaml

```diff
@@ -121,18 +118,14 @@ Resources:
             Value: !Ref AWS::AccountId
           - Name: REACT_APP_ENVIRONMENT
             Value: !Ref Environment
```

Comment:
What does the web app need account id, region, and saas boost environment for?

Reply:
Account and Region are used to construct the links for the ECR repository upload instructions, since the AppConfig only saves the ECR repository id, not a full URL.

But we don't need environment at all, nice catch.